src/app/Shared/Checkbox.component.tsx

```diff
@@ -0,0 +1,39 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import { Checkbox, FormControlLabel } from "@material-ui/core";
+
+export interface CheckboxesProps {

```

Comment:
No, I need the name, otherwise I cannot select/deselect the checkbox.

Reply:
Ok, that's how material ui handles the checkboxes internally. So you can remove it from the onChange event but you can also leave it as an optional value and for convenience and sticking to the material ui pattern.

Reply:
But I need the name in the onChangeEvent to know which Filter was selected in the checkbox. I don't quite get it.

Reply:
The idea is that this component defines only one checkbox, so the using parent component does know which checkbox is meant because it creates it. So the name is not really necessary, just syntactic sugar for the parent component.
But as discussed, just leave it as it is because it is the same pattern material ui uses.